packages/ui/src/lib/hooks/useTools.tsx

```diff
@@ -94,6 +94,21 @@ export function ToolsProvider({ overrides, children }: TLUiToolsProviderProps) {
 					trackEvent('select-tool', { source, id: 'draw' })
 				},
 			},
+			{
+				id: 'geo',
+				label: 'tool.rectangle' as TLUiTranslationKey,
+				readonlyOk: false,
+				icon: 'geo-rectangle',
+				kbd: 'g',
+				onSelect(source) {
+					editor.updateInstanceState(
+						{ propsForNextShape: { ...editor.instanceState.propsForNextShape, geo: 'rectangle' } },
+						true
+					)
+					editor.setSelectedTool('geo')
+					trackEvent('select-tool', { source, id: `geo` })
+				}
+			},
```

Comment:
I'm curious about this one: we don't currently have a specific geo tool shortcut, just shortcuts a few of the other geo tools (rectangle, ellipse). Is there a case where you'd need that general tool defined?

Reply:
Just wanted to create a tool or action for the kbd `g` mapping it to its previous ability, which was selecting the geo tool. Feel free to modify!

Reply:
My specific motivation for the PR was actually wanting to use the `g` key for a custom tool / shape that I have, but couldn't override it since 'g' wasn't bounded to a specific tool or action beforehand.

Reply:
Ah my mistake. Ok, I'm going to remove the g binding—didn't even realize we'd had that.